### PR TITLE
test: add tests for task bodies

### PR DESCRIPTION
**Pull request by athith-g**
Added tests for the previously created task bodies. Will write tests for edge cases in the future.

### CODE REVIEW

File: tests/tasks/test_tasks.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    task_state = ""
+    wait_for_task_completion()
+    return task_state
+
+
+def test_uppercase_task():
+    """Test task that outputs uppercase version of input"""
+
+    post_response = create_task(uppercase_task_body)
+    assert post_response.status_code == 200
```
Comment (uniqueg) [inline tests/tasks/test_tasks.py:48]: Consider using `pytest` for running tests. Then, consider defining `pytest` fixtures to simplify the setup stages for your tests to minimize repetition.

I have marked these two lines, but others repeat as well, so possibly those could be included as well in a fixture (or consider having multiple fixtures).

**Review by uniqueg — CHANGES_REQUESTED** [review]
Have a look at the open Sourcery AI suggestions and consider using `pytest` to run your test and using `pytest` fixtures to simplify test code.

**Comment by athith-g** [comment]
I've converted the three tests into a single test using fixtures. I'm going to add the edge cases in a different pull request (since it involves adding input files).

**Review by uniqueg — APPROVED** [review]
Fancy! I like :))

**Review by uniqueg — APPROVED** [review]
